README.md

```diff
@@ -1,72 +1,41 @@
 # Stable | Code Exercise
 
-For this exercise, you'll implement a simple web application for users to view physical mail they've recently received at a Stable facility. Once the web application is complete, you'll deploy it for anyone with the public url to view.
+For this exercise, you'll implement a simple web application for users to view physical mail they've received at a Stable facility. Once the web application is complete, your submission will be reviewed. If we decide to move forward with the interview process, you'll demo and walk through your submission with someone from the Stable engineering team.
 
 This exercise should take 2 - 4 hours to complete. 
 
-### Business Requirements
-* The web application resembles the provided mock ups
-* The web application is accessible from the public internet
-
-### Technical Requirements
-* Users can view 6 mail items per page, with a maximum of 3 mail items per row
-* Users can click through multiple pages when there are more than 6 mail items to display
-* Users can easily interact with the web application in all screen sizes
-* The mock mail item data provided in the data.json file is served to the frontend via an endpoint
+### Requirements
+* The web application resembles the provided mock-ups
+* The mock mail item data provided in the `data.json` file (in the top level of this repository) is served to the frontend via an endpoint that supports pagination and search
```

Comment:
`The mock mail item data provided in the `data.json` file (in the top level of this repository) is served to the frontend via an endpoint`

What if we remove the requirement that pagination / search is happening on the backend and leave it open to interpretation?